src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int SUPPLY_INDEX = 0;
+    private static final int BUY_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> lines = readFile(fromFileName);
+        int[] totals = processLines(lines);
+        int supplyTotal = totals[SUPPLY_INDEX];
+        int buyTotal = totals[BUY_INDEX];
+
+        writeResults(toFileName, supplyTotal, buyTotal);
+    }
+
+    private int[] processLines(List<String> lines) {
```

Comment:
using int[] as a return type is not a good idea - hard to understand what is inside